### ISSUE TITLE
On top tooltips/values are overlapping bars in bar chart

### ISSUE THREAD

**Issue by RamiJ3mli** [diogobernardino/WilliamChart#243]
Hi! As you can see in the screenshot, the tooltips/values are overlapping the bars when they are supposed to stay on top. How can i fix this?

![screenshot](https://user-images.githubusercontent.com/22471667/54684241-0497d800-4b14-11e9-91b2-b344591a865b.png)
```
        for (i in 0 until values.size) {
            val tip = Tooltip(context, R.layout.item_chart_value)
            tip.setDimensions(Tools.fromDpToPx(24f).toInt(), Tools.fromDpToPx(10f).toInt())
            tip.setVerticalAlignment(Tooltip.Alignment.BOTTOM_TOP)
            tip.findViewById<TextView>(R.id.value).text = values[i].toString()
            tip.setHorizontalAlignment(Tooltip.Alignment.CENTER)
            tip.setMargins(0, 0, 0, Tools.fromDpToPx(8f).toInt())
            tip.prepare(traktBarChart.getEntriesArea(0)[i], 0f)
            traktBarChart.showTooltip(tip, true)
        }
```
`setTopSpacing()` will make the values overlap bars' backgrounds.
Any help is greatly appreciated.

**Comment by diogobernardino** [comment]
Version 2.x has been discontinued, I just published v3.0 completely rewritten from scratch still with limited features.

**Comment by diabloo** [comment]
you can set use border value for the axis  , 
` .setAxisBorderValues(0, (max + 10))`
i used this workaround to get extra space to put the tooltip on top of the bar .
